lib/ansible/modules/database/mongodb/mongodb_user.py

```diff
@@ -216,6 +216,12 @@ def check_compatibility(module, client):
     loose_srv_version = LooseVersion(client.server_info()['version'])
     loose_driver_version = LooseVersion(PyMongoVersion)
 
+    if loose_srv_version >= LooseVersion('3.6') and loose_driver_version < LooseVersion('3.6'):
```

Comment:
other code uses `elif`. please try following style of the surrounding code.
other quirk would be that it's a lot of copy-paste, which leads to errors in future when forgetting to update all of places of copy-pasted chunk.
I'd recommend doing looping over dict instead

Reply:
done with `COMPATIBILITY_TABLE` variables

Thanks